ROS_API.md

```diff
@@ -68,7 +68,7 @@ Below is information about the physical robot API. For the simulation, topics an
 | 🤖   | 🖥️   | Topic                         | Description                                                                                                                                                                                                                                                                                                                                                               |
 | --- | --- | ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | ✅   | ✅   | `battery/battery_status`      | Mean values of both batteries will be published if the robot has two batteries. Otherwise, the state of the single battery will be published.<br/> [*sensor_msgs/BatteryState*](https://docs.ros2.org/latest/api/sensor_msgs/msg/BatteryState.html)                                                                                                                       |
-| ✅   | ❌   | `battery/charging_status`     | Battery charging status value.<br/> [*panther_msgs/ChargingStatus*](https://github.com/husarion/panther_msgs)                                                                                                                                                                                                                                                             |
+| ✅   | ❌   | `battery/charging_status`     | Battery charging status value.<br/> [*husarion_ugv_msgs/ChargingStatus*](husarion_ugv_msgs)                                                                                                                                                                                                                                                             |
```

Comment:
should we also change it so it points to the msg file instead of just the package?